### DISCUSSION TITLE
Discussion: Are text alignment issues caused by scaling worth fixing in the near term?

### DISCUSSION THREAD

**Discussion by kmgallahan** [microsoft/microsoft-ui-xaml#7467] [General]
# Discussion: Are text alignment issues caused by scaling worth fixing in the near term?

I just created an alarm in the new Alarms & Clock app for the first time. This is what I saw:

![Alarm Clock Text Alignment](https://user-images.githubusercontent.com/5379488/110211714-df6a1480-7e65-11eb-9e0a-90aa5b13d32c.png)

This was at display scaling: 100%, text scaling 150%.

Yes, everything works as expected. However, my immediate reaction is that the many alignment issues make the UI look amateurish.

All these basic controls in the screenshot show text alignment issues, some more so than others:

* TextBox
* CheckBox (also RadioButton) #1520
* ComboBox
* Button

This discussion exists to see how other developers feel about the problem.

* Maybe less than 1% of users touch scaling, so the problem isn't worth addressing?
* Maybe purely aesthetic issues aren't enough of a concern at this time, even with the major Windows refresh coming?
* As end users, how would you feel if presented with the user interface above? 
* Would you ask / want the developers to "fix" the app? 
* Would all these alignment issues compel you to look for a replacement app (read: non-WinUI / web-based)?

Related:

* #1226
* #1255
* #1520
* https://github.com/AvaloniaUI/Avalonia/pull/4631

**Comment by marcelwgn** [comment]
I thing they are still worth fixing, however right now, getting consistent good looking design is quite difficult as #1520 shows.
